agent/internal/actor/treev4.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Sqreen. All Rights Reserved.
+// Please refer to our terms for more information:
+// https://www.sqreen.io/terms.html
+
+// Radix tree
+//
+// A radix-tree is used to efficiently store security actions by IP addresses
+// and networks.
+package actor
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/kentik/patricia"
```

Comment:
do we track all added dependencies somewhere. This can introduce hasard (unpublish).

Reply:
Yes, they are automatically tracked in files [`go.mod`](./go.mod) and [`go.sum`](./go.sum), so that is the version we use. I also wrote tests of the behavior I expect from it so that I know when the package breaks [here](https://github.com/sqreen/go-agent/pull/45/commits/c4cd1eb36aa8fd1530f823fae1fa5cf0bba5b167#diff-db7f5d3abae9fd210ec1b09fd8754b92R13).

Reply:
What about someone deleting a github repo?